lib/elixir/src/elixir_parser.yrl

```diff
@@ -407,6 +408,14 @@ dot_identifier -> matched_expr dot_op identifier : build_dot('$2', '$1', '$3').
 
 dot_alias -> aliases : {'__aliases__', meta_from_token('$1', 0), ?exprs('$1')}.
 dot_alias -> matched_expr dot_op aliases : build_dot_alias('$2', '$1', '$3').
+dot_alias -> matched_expr dot_op dot_multi_alias_expr : build_dot_multi_alias('$2', '$1', '$3').
+
+dot_multi_alias_expr -> open_curly '}' : [].
+dot_multi_alias_expr -> open_curly dot_multi_alias_args close_curly : '$2'.
+
+dot_multi_alias_args -> matched_expr : ['$1'].
```

Comment:
But that's the point. :) We should not treat `x.{y, z, w}` as anything special at the parser level.